packages/connect-core/src/entities/Organization.ts

```diff
@@ -86,7 +86,11 @@ export default class Organization {
   ///////// APPS ///////////
   async apps(): Promise<Application[]> {
     this.checkConnected()
-    return this._connector.appsForOrg(this.address)
+    const apps = await this._connector.appsForOrg(this.address)
+    for (const app of apps) {
+      await app._init()
```

Comment:
I wonder what you think of handling here like this. I was not sure if a `Promise.all` was a better pattern in this case.

Reply:
I think it may be better to handle them using `Promise.all` as they will be able to be resolved in parallel rather than sequentially. 😃